pkg/virt-controller/watch/migration.go

```diff
@@ -687,6 +693,34 @@ func (c *MigrationController) expandPDB(pdb *policyv1.PodDisruptionBudget, vmi *
 	return nil
 }
 
+func (c *MigrationController) handleMigrationBackoff(key string, vmi *virtv1.VirtualMachineInstance, migration *virtv1.VirtualMachineInstanceMigration, targetPod *k8sv1.Pod) error {
+	id := string(vmi.UID)
+
+	switch migration.Status.Phase {
+	case virtv1.MigrationSucceeded:
+		c.backoffHandler.DeleteEntry(id)
+		log.Log.V(4).Infof("migration %s completed, deleting backoff entry", migration.Name)
+	case virtv1.MigrationPreparingTarget, virtv1.MigrationTargetReady:
```

Comment:
Do we need this case? Don't we cycle through `MigrationFailed` eventually?

Reply:
I put this for a reason, but I don't remember honestly, let me try it, I had different versions while I was working on this so you might be right.